internal/modules/module.go

```diff
@@ -195,6 +208,19 @@ func (m *Module) GetFS(ctx context.Context) (billy.Filesystem, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed to clone module: %w", err)
 		}
+
+		// Because this is a new feature, don't fail if we can't open the
+		// repo for now. If we have a commit, ensure it matches the commit
+		// we expect.
+		if m.Version.Commit != "" {
```

Comment:
The goal here is that eventually if the `commit` changes out from under the user, I want to have `stencil upgrade` to have to be ran.